pkg/build/controller/strategy/util.go

```diff
@@ -17,7 +17,7 @@ const (
 	sourceSecretMountPath     = "/var/run/secrets/openshift.io/source"
 )
 
-var whitelistEnvVarNames = []string{"BUILD_LOGLEVEL"}
+var whitelistEnvVarNames = []string{"BUILD_LOGLEVEL", "HTTP_PROXY", "HTTPS_PROXY", "http_proxy", "https_proxy"}
```

Comment:
Can we type this once and assume we'll always allow both upper case and lower case vars?